cmd/nomos/parse/find_files.go

```diff
@@ -32,7 +32,7 @@ import (
 func FindFiles(dir cmpath.Absolute) ([]cmpath.Absolute, error) {
 	out, err := exec.Command("find", dir.OSPath(), "-type", "f", "-not", "-path", "*/\\.git/*").CombinedOutput()
 	if err != nil {
-		return nil, errors.Wrap(err, string(out))
+		return nil, fmt.Errorf("%s: %w", out, err)
```

Comment:
Still need the string cast for byte array. The %s will see it as an array and try to stringify the array instead of casting. This is a pretty common gotcha and might be elsewhere too.

Reply:
Good catch.  Fixed some of them and will need to think where else my regex replaced them.

Reply:
Run git diff and grep for "string("

Reply:
I think we are good.  Here is the git diff output:
```
λ git diff main mb_err_pkg | grep "string("
-               return nil, errors.Wrap(err, string(out))
+               return nil, fmt.Errorf("%s: %w", string(out), err)
        files := strings.Split(string(out), "\n")
-                               t.Fatal(errors.Wrap(err, string(out)))
+                               t.Fatal(fmt.Errorf("%s: %w", string(out), err))
-                                               t.Fatal(errors.Wrap(err, string(out)))
+                                               t.Fatal(fmt.Errorf("%s: %w", string(out), err))
-                                       t.Fatal(errors.Wrap(err, string(out)))
+                                       t.Fatal(fmt.Errorf("%s: %w", string(out), err))
                        name, err, string(out))
        operations := strings.Fields(string(out))
                        operation, err, string(out))
-                       return errors.Errorf("failed to delete cluster %s: %v\nstdout/stderr:\n%s", name, err, string(out))
+                       return fmt.Errorf("failed to delete cluster %s: %v\nstdout/stderr:\n%s", name, err, string(out))
-               return errors.Errorf("failed to create cluster %s: %v\nstdout/stderr:\n%s", name, err, string(out))
+               return fmt.Errorf("failed to create cluster %s: %v\nstdout/stderr:\n%s", name, err, string(out))
-               return errors.Errorf("failed to get credentials: %v\nstdout/stderr:\n%s", err, string(out))
+               return fmt.Errorf("failed to get credentials: %v\nstdout/stderr:\n%s", err, string(out))
-               return errors.Errorf("failed to refresh access_token: %v\nstdout/stderr:\n%s", err, string(out))
+               return fmt.Errorf("failed to refresh access_token: %v\nstdout/stderr:\n%s", err, string(out))
                        clusterName, err, string(out))
        clusters := strings.Fields(string(out))
        match := re.FindStringSubmatch(string(bytes))
-               return "", errors.Wrap(err, string(out))
+               return "", fmt.Errorf("%s: %w", string(out), err)
        if strings.Contains(string(out), "\"type\": \"error\"") {
                return "", errors.New(string(out))
-                       errs = multierr.Append(errs, errors.Wrap(err, string(out)))
+                       errs = multierr.Append(errs, fmt.Errorf("%s: %w", string(out), err))
                        errs = multierr.Append(errs, errors.New(string(out)))
-               return "", errors.Wrap(err, string(out))
+               return "", fmt.Errorf("%s: %w", string(out), err)
-               return "", errors.Wrap(err, string(out))
+               return "", fmt.Errorf("%s: %w", string(out), err)
        return string(out), nil
-               return -1, errors.Wrap(err, string(out))
+               return -1, fmt.Errorf("%s: %w", string(out), err)
-               return "", errors.Wrap(err, string(out))
+               return "", fmt.Errorf("%s: %w", string(out), err)
        if !strings.Contains(string(out), fmt.Sprintf("\"name\":\"%s\"", repoName)) {
                return "", errors.New(string(out))
-               return "", errors.Wrap(err, string(out))
+               return "", fmt.Errorf("%s: %w", string(out), err)
-                               errs = multierr.Append(errs, errors.Wrap(err, string(out)))
+                               errs = multierr.Append(errs, fmt.Errorf("%s: %w", string(out), err))
                        if !strings.Contains(string(out), "\"message\":\"202 Accepted\"") {
-                       errs = multierr.Append(errs, errors.Wrap(err, string(out)))
+                       errs = multierr.Append(errs, fmt.Errorf("%s: %w", string(out), err))
                if !strings.Contains(string(out), "\"message\":\"202 Accepted\"") {
-                       return errors.Errorf("unexpected response in DeleteRepoByID: %s", string(out))
+                       return fmt.Errorf("unexpected response in DeleteRepoByID: %s", string(out))
-                       return nil, errors.Wrap(err, string(out))
+                       return nil, fmt.Errorf("%s: %w", string(out), err)
-               return errors.Errorf("failed to copy directory: %s", string(out))
+               return fmt.Errorf("failed to copy directory: %s", string(out))
        return strings.TrimSpace(string(out)), nil
                if string(actual) != value {
-               return out, errors.Wrapf(err, "invoking helm: %s", string(out))
+               return out, fmt.Errorf("invoking helm: %s: %w", string(out), err)
        return string(bytes), nil
                newValue := string(newObj.Data[string(configsync.AuthSSH)])
mikebz-ct src/kpt-config-sync λ 
```